siafile.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
+type Siafile struct {
+	Id           bson.ObjectId `bson:"_id,omitempty", json:"_id"`
```

Comment:
Object Id's aren't a real json thing. They are a special object. Mongo uses them for ObjectId's http://docs.mongodb.org/manual/reference/object-id/